<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_screenshoterror</class>
 <widget class="QWidget" name="w_screenshoterror">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxSaveError">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>790</y>
     <width>951</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="comboBoxErrorType">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtCommentaire">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Image</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnClearPainter">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>790</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear painter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>830</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnColor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>790</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="boxColor">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>830</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbComment">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>850</y>
     <width>661</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
